src/assets/compendiums/name-bases.db/name-bases.yaml

```diff
@@ -0,0 +1,77 @@
+name: 'Names: Sartarite, Female'
```

Comment:
Can we put flags onto these JournalEntry items here in the yaml?  That way we can flag the rqid.

`{
    "rqg": {
        "rqid": "je.name-bases-names-sartarite-female",
        "rqidPriority": 0,
        "rqidLang": "en"
    },
    "core": {
        "sourceId": "Compendium.rqg.name-bases.JNQLdEHBRqoXFcMx"
    }
}`